chat_client/static/js/klatchatNano.js

```diff
@@ -2191,7 +2208,145 @@ if (isAudio === '0'){
 textInputElem.value = "";
 }
 }
-}/**
+}
+const myAccountLink = document.getElementById('myAccountLink');
+const settingsLink = document.getElementById('settingsLink');
+
+/**
+* Shows modal associated with profile
+* @param nick: nickname to fetch
+* @param edit: to open modal in edit mode
+*
+* @return true if modal shown successfully, false otherwise
+*/
+async function showProfileModal(nick=null, edit='0'){
+let fetchURL = `${configData['currentURLBase']}/components/profile?`
+let modalId;
+let avatarId;
+if(edit === '1'){
+modalId = `${currentUser['nickname']}EditModal`;
+// avatarId = `${currentUser['nickname']}EditAvatar`;
+fetchURL += `edit=1`;
+}else{
+modalId = `${nick}Modal`;
+// avatarId = `${nick}Avatar`;
+fetchURL += `nickname=${nick}`;
+}
+const profileModalHTML = await fetch(fetchURL, {headers: new Headers({'Authorization': getSessionToken()})}).then(async (response) => {
+if (response.ok) {
+return await response.text();
+}
+throw `unreachable (HTTP STATUS:${response.status}: ${response.statusText})`
+}).catch(err=> {
+console.warn(err);
+return null;
+});
+if (profileModalHTML){
+const existingModal = document.getElementById(modalId);
+deleteElement(existingModal);
+const main = document.getElementById('main');
+main.insertAdjacentHTML('afterbegin', profileModalHTML);
+const existingModalJQuery = $(`#${modalId}`);
+existingModalJQuery.modal('show');
+return true
+} return false;
+}
+
+/**
+* Convenience wrapper to show modal in the edit mode
+*/
+async function showProfileEditModal(){
+return await showProfileModal(null, '1');
+}
+
+/**
+* Previews uploaded image
+* @param nickname: target nickname
+*/
+const previewFile = (nickname) => {
+const userNewAvatar = document.getElementById(`${nickname}NewAvatar`);
+const userEditAvatar = document.getElementById(`${nickname}EditAvatar`);
+if (userNewAvatar?.files.length > 0) {
+const objectURL = window.URL.createObjectURL(userNewAvatar.files[0]);
+try{
+URL.revokeObjectURL(userEditAvatar.src);
+} catch (e) {
+console.debug('Its initial URL');
+}
+userEditAvatar.src = objectURL;
+}
+}
+
+async function initProfileEditModal(){
```

Comment:
Consider adding a docstring for this function. If there's an easy way to reformat indentation for this file, it would be easier to read

Reply:
done in https://github.com/NeonGeckoCom/pyklatchat/pull/57/commits/c660e3aa9a848f1c9a61815beb1a2e411dc5243a

> Consider adding a docstring for this function. If there's an easy way to reformat indentation for this file, it would be easier to read